frontend/src/locales/es.json

```diff
@@ -370,7 +370,7 @@
     "Are there trade limits?": "¿Hay límites de intercambios?",
     "Maximum single trade size is {{maxAmount}} Satoshis to minimize lightning routing failure. There is no limits to the number of trades per day. A robot can only have one order at a time. However, you can use multiple robots simultaneously in different browsers (remember to back up your robot tokens!).": "Para minimizar fallos en el enrutamiento Lightning, el máximo por intercambio es de {{maxAmount}} Sats. No hay límite de intercambios en el tiempo. Aunque un Robot solo puede intervenir en una orden a la vez, puedes usar varios Robots en diferentes navegadores (¡recuerda guardar los tokens de tus Robots!).",
     "Is RoboSats private?": "¿RoboSats es privado?",
-    "RoboSats will never ask you for your name, country or ID. RoboSats does not custody your funds and does not care who you are. RoboSats does not collect or custody any personal data. For best anonymity use Tor Browser and access the .onion hidden service.": "RoboSats no custodia tus fondos y no recolecta o custodia ningún dato personal, pues no le importa quien eres. RoboSats nunca te preguntará por tu nombre, país o número de documento. Para mejorar tu privacidad, usa el Navegador Tor y visita el sitio cebolla.",
+    "RoboSats will never ask you for your name, country or ID. RoboSats does not custody your funds and does not care who you are. RoboSats does not collect or custody any personal data. For best anonymity use Tor Browser and access the .onion hidden service.": "RoboSats no custodia tus fondos y no recolecta o custodia ningún dato personal, pues no le importa quien eres. RoboSats nunca te preguntará por tu nombre, país o número de documento. Para mejorar tu privacidad, usa el Navegador Tor y accede al enlace .onion.",
```

Comment:
`sitio cebolla` is the spanish Tor Project official translation of Onion services / .onion site

Source https://tb-manual.torproject.org/es/onion-services/

Reply:
Yes, you are right. I just proposed this alternative because not everybody knows what "servicio cebolla" means, while  "enlace .onion" seems something easier to understand.

Reply:
I think it does not matter which one we pick:) 

Sticking to the Tor Project terminology is a plus. But I had never heard "sitio cebolla" before you proposed it.The Tor Project probably does not have any linguistic professional on their team either :D. 

Onion is also straight forward to understand in Spanish since URLs finish with ".onion". It seems to be the most used term by the Spanish speakers in the wild.

Reply:
I vote in this case for `sitio .onion`! I will contact also to Tor Project and see... Cheers!